packages/falcon-sensor/generated-changes/overlay/app-readme.md

```diff
@@ -1,9 +0,0 @@
-# CrowdStrike Falcon
```

Comment:
overlaid app-readme.md == upstream app-readme.md in the chart that the URL points to, so this was not necessary and deleted.